### DISCUSSION TITLE
手写数学公式识别算法-CAN-使用样例代码也无法识别

### DISCUSSION THREAD

**Discussion by charliedream1** [PaddlePaddle/PaddleOCR#12187] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu 22
- 版本号/Version：Paddle：Realease 2.7  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
  -  使用algorithm_rec_can.md中的样例脚本跑测试
  -  python3 tools/infer_rec.py -c configs/rec/rec_d28_can.yml -o Architecture.Head.attdecoder.is_train=False Global.infer_img='./doc/datasets/crohme_demo/hme_00.jpg' Global.pretrained_model=./rec_d28_can_train/best_accuracy.pdparams
- 完整报错/Complete Error Message：
  - ```bash
   [2024/05/13 20:09:06] ppocr INFO: load pretrain successful from /data/nvme0/mdl/mdl_zoo/ocr/rec_d28_can_train/best_accuracy
[2024/05/13 20:09:06] ppocr INFO: infer_img: ./doc/datasets/CDLA_demo/val_0633.jpg
[2024/05/13 20:09:07] ppocr INFO:        result:        []
[2024/05/13 20:09:07] ppocr INFO: success!
``` 
输出的result为什么是空呢？

**Comment by UserWangZz** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/47658498/b07c6ece-3758-4cf0-bdb4-bc5bb5edfd9f)
你好，这边复现，没有发现问题，需要你再检查一下

**Comment by charliedream1** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/15007828/319a479c-2976-4355-801c-b14d5014a66b)
请问一下，可能的原因会是什么呢？还是出不来

**Comment by charliedream1** [comment]
我安装的版本：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/15007828/2ebf5f08-a8d5-4da0-9de0-5ffc525a9a1f)

**Comment by UserWangZz** [comment]
Global.checkpoints=./rec_d28_can_train/best_accuracy
改成这样试试

**Comment by UserWangZz** [comment]
看你执行的命令，目前没有发现有什么问题

**Comment by charliedream1** [comment]
> Global.checkpoints=./rec_d28_can_train/best_accuracy 改成这样试试

改完还是没有效果

**Comment by UserWangZz** [comment]
尝试使用cpu推理一下看看

**Comment by charliedream1** [comment]
使用cpu的传参是什么，没有找到，直接写use_gpu=False？

**Comment by UserWangZz** [comment]
是的

**Comment by UserWangZz** [comment]
你好，请问使用cpu推理后，是否可以得到正常结果呢？
